### DISCUSSION TITLE
Can FIDI's default container port be changed from 8080 to 8081?

### DISCUSSION THREAD

**Discussion by zhanghengtian** [firefly-iii/firefly-iii#5875] [Firefly III]
As I choosed to set the fireflyiii and FIDI's network setting just same as the local synogloy's host, by doing this, it make me easy to connect my own mysql database. 

Therefore it will use the same IP range with synology. but the fireflyiii app and FIDI used same port :8080. which will lead to error. bcos they can not work in the same port for two docker app.

Can FIDI's default container port be changed from 8080 to 8081?
Appericated for answers!

**Comment by JC5** [comment]
Yes, you can configure Docker to forward 8080 to 8081. I’m not sure how synology does this, but the fact that Docker containers use the same ports doesn’t have to be a problem for the host.
